src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -218,10 +218,16 @@ internal abstract partial class BaseArgument : Operation, IArgumentOperation
         /// </summary>
         public IParameterSymbol Parameter { get; }
         protected abstract IOperation ValueImpl { get; }
-        internal IConvertibleConversion InConversionConvertible { get; }
-        internal IConvertibleConversion OutConversionConvertible { get; }
-        public CommonConversion InConversion => InConversionConvertible.ToCommonConversion();
-        public CommonConversion OutConversion => OutConversionConvertible.ToCommonConversion();
+        internal IConvertibleConversion InConversionConvertibleOpt { get; }
+        internal IConvertibleConversion OutConversionConvertibleOpt { get; }
+        public CommonConversion InConversion => InConversionConvertibleOpt?.ToCommonConversion() ?? Identity();
+        public CommonConversion OutConversion => OutConversionConvertibleOpt?.ToCommonConversion() ?? Identity();
+
+        private static CommonConversion Identity()
+        {
+            return new CommonConversion(exists: true, isIdentity: true, isNumeric: false, isReference: false, methodSymbol: null);
```

Comment:
We should just make this a shared static.

Reply:
> We should just make this a shared static.

CommonConversion is a pretty simple structure, I do not believe there would be a benefit from creating a field to cache this value.